### PR TITLE
Stream line using of List of volumes and single volume

### PR DESCRIPTION
**Pull request by bostko**
- customizers use multiple volumes
- effector use single volume

### CODE REVIEW

**Comment by bostko** [comment]
Fixes customizer tests which expect different type of arguments.

File: blockstore/src/main/java/brooklyn/location/blockstore/api/VolumeOptions.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package brooklyn.location.blockstore.api;
+
+import brooklyn.location.blockstore.BlockDeviceOptions;
+import brooklyn.location.blockstore.FilesystemOptions;
+import org.apache.brooklyn.util.core.flags.TypeCoercions;
+
+import java.util.Map;
+
+public class VolumeOptions {
```
Comment (iyovcheva) [inline blockstore/src/main/java/brooklyn/location/blockstore/api/VolumeOptions.java]: Good idea to create type for that

File: blockstore/src/main/java/brooklyn/location/blockstore/BlockDeviceOptions.java [hunk 2]
```diff
@@ -27,17 +28,24 @@ public static BlockDeviceOptions fromMap(Map<String, ?> map) {
             }
         }
         if (map.containsKey("sizeInGb")) {
+            checkNotNull(map.get("sizeInGb"), "sizeInGb should not be null");
             if (map.get("sizeInGb") instanceof Double && Math.abs((Double) map.get("sizeInGb") - ((Double) map.get("sizeInGb")).intValue()) >= 0.01
                     || map.get("sizeInGb") instanceof Float && Math.abs((Float) map.get("sizeInGb") - ((Float) map.get("sizeInGb")).intValue()) >= 0.01) {
                 throw new UnsupportedOperationException("Trying to set block device with not allowed sizeInGb value "
                         + map.get("sizeInGb") + "; sizeInGb must have integer value.");
             } else if (map.get("sizeInGb") instanceof Double) {
-                result.sizeInGb = checkNotNull(((Double) map.get("sizeInGb")).intValue(), "sizeInGb");
```
Comment (iyovcheva) [inline blockstore/src/main/java/brooklyn/location/blockstore/BlockDeviceOptions.java]: Why removed the `checkNotNull` check?

**Comment by iyovcheva** [comment]
Looks good.

**Pull request merged**